extensions/standard-processors/processors/LogAttribute.cpp

```diff
@@ -39,12 +39,14 @@ namespace nifi {
 namespace minifi {
 namespace processors {
 
-core::Property LogAttribute::LogLevel(core::PropertyBuilder::createProperty("Log Level")->withDescription("The Log Level to use when logging the Attributes")->withAllowableValues<std::string>({
+core::Property LogAttribute::LogLevel(core::PropertyBuilder::createProperty("Log Level")->withDescription("The Log Level to use when logging the Attributes")->withAllowableValues<std::string>( {
     "info", "trace", "error", "warn", "debug" })->build());
 
 core::Property LogAttribute::AttributesToLog(
     core::PropertyBuilder::createProperty("Attributes to Log")->withDescription("A comma-separated list of Attributes to Log. If not specified, all attributes will be logged.")->build());
 
+core::Property LogAttribute::FlowFilesToLog(core::PropertyBuilder::createProperty("FlowFiles To Log")->withDescription("Number of flow files to log")->withDefaultValue<int>(1)->build());
```

Comment:
Why not uint? 
We have unsigned validator, too. 
That would prevent negative and 0 could mean "log as many as available in the session".

Reply:
No clue what I was thinking on the not using the UNSIGNED_LONG_VALIDATOR. 

Using zero for that purpose was something I wavered on since that can break the contract of scheduling, but I'll add a solid warning to the description. It will be useful for testing.